oneview_redfish_toolkit/api/computer_system.py

```diff
@@ -143,3 +146,31 @@ def map_boot(self, boot_list):
             redfish_boot_list.append('None')
 
         return redfish_boot_list
+
+    def _fill_resource_block_members(self, server_profile, drives):
+        resource_block_uuids = \
+            self._get_resource_block_uuids(server_profile, drives)
+
+        resource_blocks_base_uri = \
+            "/redfish/v1/CompositionService/ResourceBlocks/{}"
```

Comment:
We should use `ResourceBlock.BASE_URI` instead of "/redfish/v1/CompositionService/ResourceBlocks".